Feeds/Startup.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Etch.OrchardCore.Lever.Api.Services;
+using Etch.OrchardCore.Lever.Services;
+using Microsoft.Extensions.DependencyInjection;
+using OrchardCore.Modules;
+
+namespace Etch.OrchardCore.Lever.Feeds
+{
+    [Feature("Etch.OrchardCore.Lever.Feeds")]
+    public class Startup : StartupBase
+    {
+        public override void ConfigureServices(IServiceCollection services)
+        {
+            services.AddMvc();
+            services.AddHttpClient();
```

Comment:
Out of curiosity, how comes `AddHttpClient` is needed?

Reply:
This was probably added by mistake.

Reply:
Actually, I looked into this again and it's needed by PostingApiService which is a dependency for LeverPostingService and both are in the startup.